enhancements/installer/telemeter-metrics.md

```diff
@@ -0,0 +1,300 @@
+---
+title: installer-telemetry-metrics
+authors:
+  - "@patrickdillon"
+  - "@rna-afk"
+reviewers:
+  - "@abhinavdahiya"
+  - "@brancz"
+  - "@crawford"
+  - "@sdodson"
+approvers:
+  - "@abhinavdahiya"
+  - "@brancz"
+  - "@crawford"
+  - "@sdodson"
+creation-date: 2020-02-07
+last-updated: 2020-02-07
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Installer Telemetry Metrics
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+ > 1. This requires creating an [aggregation pushgateway][weaveworks-aggregation-pushgateway] where the installer would push metrics. How would this be handled and by which team?
+ > 2. If #1 is possible, how would client authentication be administered to prevent [malicious pushing of metrics](risks-and-mitigations)? Authentication based on what? A pull secret? 
+ > 3. How to filter to collect only real user usage and prevent OpenShift developer/Red Hat internal users from skewing metrics? Perhaps filter by version?
+ > 4. Is it possible to determine whether a user modified an ignition config or manifest asset by comparing the asset the installer would generate to the one found on disk?
+ > 5. Are [package-scoped variables](internal-installer-design) acceptable in the proposed installer `metrics` package? 
```

Comment:
Could you add more detail here? It's hard for me to tell whether this has anything to do with the actual metrics we send (I care about this) or internals of the code (I don't particularly care about :) ).